aries-site/src/pages/templates/lists.mdx

```diff
@@ -187,8 +187,20 @@ With long sets of data, pagination can help to create a more efficient means of
 
 ### Item order
 
-Allowing the user to re-order the list by dragging items or pressing move up and down controls. Choose `Actions->Re-Order Items` to
-enable drag and drop of list items and move up/down controls. Press `Done` when finished.
+For **multi-line** list items, they should be set to top aligned for easy scanning across a row.
```

Comment:
Docs wording suggestion:

List supports `onOrder` property which allows the user to re-order the list by dragging items or pressing up and down controls.

Choose `Actions -> Re-Order Items` to enable drag and drop of list items and move up/down controls. While in re-ordering mode, take note of how the row content is top-aligned. For **multi-line** list items like this use case, list items should be top-aligned for easy scanning across a row.

To implement this, use the prop `defaultItemsProp` on List:

```
<List
defaultItemsProp={{align: 'start'}}
...any additional props
<List/>
```